tools/monitoring/landsat_gap_filler.py

```diff
@@ -1 +1,198 @@
-print("Here for future implementation")
+"""
+# Read report and generate messages to fill missing scenes
+
+"""
+import json
+import logging
+import sys
+import traceback
+from typing import Optional
+
+import click
+from odc.aws.queue import publish_messages, get_queue
+
+from tools.utils.utils import (
```

Comment:
Importing from utils.utils is a bit silly! It would have been better to have the utils.py file in the `tools` folder.

Reply:
This Utils might get too big... actually I reckon it's already a bit confusing..... My idea is soon to refactor that and split into specific utils